src/engine/Map.cxx

```diff
@@ -66,21 +66,12 @@ NeighborNodesPosition operator++(NeighborNodesPosition &nn, int)
   return res;
 }
 
-void Map::getNodeInformation(const Point &isoCoordinates) const
+Map::Map(int columns, int rows)
```

Comment:
No, the function is needed to create an empty map without any mapnodes. 
The other constructor creates either terrain gen nodes (with height information, different terrain tiles, trees, ....) or nodes with just default terrain (terrain_grass).
But when we load a game, we need a map object without any nodes. those are recreated after parsing the save game.
The only alternative would be to always create an empty map in the constructor and optionally call a generateTerrain() or generatePlainMap() function. 
But I rather dislike this option because recreating the map requires a teardown (destroying and recreating the object) beforehand to make sure everything is in order.